packages/vue-vuetify/src/complex/ObjectRenderer.vue

```diff
@@ -10,13 +10,14 @@
       :cells="control.cells"
     />
     <additional-properties
-      v-if="hasAdditionalProperties && showAdditionalProperties"
+      v-if="hasAdditionalProperties || showAdditionalProperties"
```

Comment:
If `showAdditionalProperties === false` then we should still hide them, right?

Reply:
that is more like show additional properties even when additionalProperties is missing - this is because by jsonschema spec the default value is to allow additional properties while here in this implementation that needs to be explicitly set to true or an object - e.g. missing additionalProperties won't cause the object renderer to render the additional properties UI but if you specify that in the UI schema then it will